lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -86,6 +86,10 @@ public function get_items_permissions_check( $request ) {
 			return new WP_Error( 'rest_forbidden_context', __( 'Sorry, you cannot view this resource with edit context.' ), array( 'status' => rest_authorization_required_code() ) );
 		}
 
+		if ( in_array( $request['orderby'], array( 'email' ), true ) && ! current_user_can( 'list_users' ) ) {
```

Comment:
Ah, yes we should add `registered_date` in that case, we shouldn't be exposing this right now.